src/main/java/org/pegdown/Extensions.java

```diff
@@ -147,10 +147,15 @@
     static final int TASKLISTITEMS = 0x00200000;
 
     /**
+     * GitHub style task list items: - [ ] and - [x]
```

Comment:
It is right in the readme, line 32:

* EXTANCHORLINKS uses all text of the header for ANCHORLINKS, spaces and non-alphanumerics replaced by `-`, multiple dashes trimmed to one.

Feel free to fix the comment.